ownCloud/Client/ClientQueryViewController.swift

```diff
@@ -147,6 +147,9 @@ class ClientQueryViewController: UITableViewController, Themeable {
 		self.tableView.dragInteractionEnabled = true
 
 		let actionsBarButton: UIBarButtonItem = UIBarButtonItem(barButtonSystemItem: .add, target: self, action: #selector(uploadsBarButtonPressed))
+		actionsBarButton.accessibilityLabel = "Upload files".localized
+		actionsBarButton.setTitleTextAttributes([.font :UIFont.systemFont(ofSize: 10)], for: .normal)
```

Comment:
@pablocarmu Couldn't we use` UIFont.preferredFont(forTextStyle:)` here?

Reply:
After re-reading the Apple's accessibility guidelines again I've noticed that the navigation/toolbar items don't scale their font size like regular labels. Instead, you can provide a PDF vector inside an asset catalog and, when you long press on these elements, an alert-style description is provided (vector image + bigger message).

I think we can make this in the second iteration of this feature because we need to collect all the SVGs we're using and convert them to PDFs